lms/templates/verify_student/make_payment_step.underscore

```diff
@@ -30,7 +30,7 @@
     <div class="instruction <% if ( !upgrade && isActive ) { %>center-col<% } %>">
       <% if ( _.some( requirements, function( isVisible ) { return isVisible; } ) ) { %>
       <p class="instruction-info">
-        <% if ( verificationDeadline ) { %>
+        <% if ( verificationDeadline && isActive) { %>
```

Comment:
@mushtaqak The change is made when `isActive` is `True`, I think we should have a test where `isActive` is `True`. 
The PR contains test with isActive set to false.

Is there somthing i am missing?